x-pack/plugins/enterprise_search/server/routes/app_search/documents.test.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { MockRouter, mockRequestHandler, mockDependencies } from '../../__mocks__';
+
+import { registerDocumentRoutes } from './documents';
+
+describe('document routes', () => {
+  describe('GET /api/app_search/engines/{engineName}/documents/{documentId}', () => {
+    let mockRouter: MockRouter;
+
+    beforeEach(() => {
+      jest.clearAllMocks();
+      mockRouter = new MockRouter({
+        method: 'get',
+        path: '/api/app_search/engines/{engineName}/documents/{documentId}',
+        payload: 'query',
```

Comment:
[nit] This just occurred to me, but if we're not using schema validation I think we can probably just leave off the `payload` key entirely since that's all it's used for

```suggestion
```

Reply:
Oh great, I didn't realize it was optional.